packages/vue-components/src/panels/PanelBase.js

```diff
@@ -53,6 +53,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    showPreview: {
```

Comment:
Doesn't that mean the user will have to indicate "peek" instead of "show-preview" to use the preview feature?

Reply:
yup, suggesting a user side change here (less to type)

Reply:
Hmm, I'm thinking that it may be a little confusing for the user. If we were to go ahead with this, then I think perhaps we should package/name the feature as "peek" instead of "preview" (that means in the user guide we will strictly use "peek" to refer to this feature). 

However, I personally think "preview" sounds better but this is very subjective so I am open to both :-)

Reply:
> we should package/name the feature as "peek" instead of "preview" 

yup.

or just `preview` would be good too (vs `show-preview`)

I'm open to either as well, maybe we can get a user opinion here? 🙂
@damithc

Reply:
I'll shorten `show-preview` to `preview` in the meantime, while waiting for Prof's opinion.

Reply:
I'm OK with either `preview` or `peek`. I thought `peek` is a better fit because it feels like _showing only a bit_ but I could be wrong. Also, the word _preview_ might be used in other contexts, e.g., _live preview_ of pages via the `markbind serve` command.

Reply:
Hmm, considering that the word "preview" is used in other contexts, now I am leaning towards "peek". 

I'll change it to "peek" then :-)